### DISCUSSION TITLE
Account compromised by other user and used for scammed

### DISCUSSION THREAD

**Discussion by Unknown8877** [community/community#130827] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

All my accounts was hacked by one of the users of GitHub and was used to scammed. Also violated the law plagiarism. Hopping for someone who could help as no one could 😭

**Comment by Akash1134** [comment]
Hi @Unknown8877 , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
